### PR TITLE
New branch without Coverage report

### PR DESCRIPTION
**Pull request by Benjamin-Cardon**

### CODE REVIEW

File: spec/questionsSpec/ModalSpec/AddQuestionModal.spec.jsx
Comment (lundas) [inline spec/questionsSpec/ModalSpec/AddQuestionModal.spec.jsx:1]: looks like you're mixing ES6 and CommonJS imports on line 7. particular reason for that?

File: spec/questionsSpec/NavigationSpec/NavigationButtons.spec.jsx
Comment (lundas) [inline spec/questionsSpec/NavigationSpec/NavigationButtons.spec.jsx:1]: mixing import styles again on line 6

File: spec/questionsSpec/NavigationSpec/SearchBar.spec.jsx
Comment (lundas) [inline spec/questionsSpec/NavigationSpec/SearchBar.spec.jsx:1]: mixing import statements again on line 6

File: spec/questionsSpec/QuestionListSpec/Answer.spec.jsx
Comment (lundas) [inline spec/questionsSpec/QuestionListSpec/Answer.spec.jsx:1]: mixing import statements again on line 7

**Review by Dillon-Armstrong — COMMENTED** [review]
Got a lot done!

File: spec/questionsSpec/QuestionListSpec/AnswerList.spec.jsx
Comment (lundas) [inline spec/questionsSpec/QuestionListSpec/AnswerList.spec.jsx:1]: mixing imports again on line 7

File: spec/questionsSpec/QuestionListSpec/Question.spec.jsx
Comment (lundas) [inline spec/questionsSpec/QuestionListSpec/Question.spec.jsx:1]: mixing imports again on line 7

File: spec/questionsSpec/Questions.spec.jsx
Comment (lundas) [inline spec/questionsSpec/Questions.spec.jsx:1]: mixed imports. do you use the axios import?

**Review by lundas — COMMENTED** [review]
fix mixed imports in spec files. remove axios import from Questions.spec if not needed. Remove comments/pseudocode unless saving for future implementation

**Review by lundas — APPROVED** [review]
looks good. thanks for making changes!

**Pull request merged**